docs/dev.rst

```diff
@@ -1,20 +1,522 @@
-Develop a project
+Develop a Project
 =================
 
-Set up a dev org
+A general overview on how to develop a Salesforce project with CumulusCI.
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
+
+.. note:: Run ``cci flow info dev_org`` for a full list of the ``dev_org`` flow steps.
+
+To run the ``dev_org`` flow against the project's `default org<TODO>`_:
+
+.. code-block:: console
+
+    $ cci flow run dev_org
+
+You can also use the ``--org`` option to explicitly list the org.
+
+    Example: Run the ``dev_org`` flow against the org currently defined as ``dev`` in CumulusCI.
+
+.. code-block:: console
+
+    $ cci flow run dev_org --org dev
+
+..
+
+    Open the new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+Set the Capture State
+^^^^^^^^^^^^^^^^^^^^^
+
+To make changes to capture in an org, use the ``snapshot_changes`` task to create a snapshot. A snapshot tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
+
+.. code-block:: console
+
+    $ cci task run snapshot_changes --org dev
+
+.. note:: There's no need to run ``snapshot_changes`` if you've set up a scratch org with any of these flows:
+
+    * ``dev_org``
+    * ``dev_org_namespaced``
+    * ``qa_org``
+    * ``regression_org``
+    * ``install_beta``
+    * ``install_prod``
+    
+    These flows all run ``snapshot_changes`` as their last step.
+
+The ``list_changes`` and ``retrieve_changes`` tasks detect new metadata but ignore any prior changes.
+
+To check that the snapshot was created successfully:
```

Comment:
So ``list_changes`` will check if a snapshot is created successfully? I ask because I don’t understand how ``list_changes`` bridges to ``snapshot_changes``.

Reply:
Hopefully my above comment helps. I think we could potentially spell this out a little more for users. If you just created a snapshot then running `cci task run list changes --org dev` should output "Found no changes." I think we could also change this first sentence:
```suggestion
The ``list_changes`` and ``retrieve_changes`` tasks detect new or modified metadata in an org but ignore any changes to metadata made prior to the last snapshot.

To check that the snapshot was created successfully you should see the following output "Found no changes":
```

Reply:
@Br4nd0R Can you send me the full text of what happens after the ``cci task run list_changes --org dev`` is run successfully? I want to put that inside the code block featured under these sentences.

Reply:
```
❯ cci task run list_changes --org dev
2021-02-09 12:16:45: Getting org info from Salesforce CLI for test-blwe3dkdxcyr@example.com
2021-02-09 12:16:49: Beginning task: ListChanges
2021-02-09 12:16:49: As user: test-blwe3dkdxcyr@example.com
2021-02-09 12:16:49: In org: 00D1F000000Mrmr
2021-02-09 12:16:49:
2021-02-09 12:16:50: Found no changes.
```